src/FakeItEasy.dotnet/project.json

```diff
@@ -18,8 +18,6 @@
   },
 
   "frameworks": {
-    "netstandard1.6": {
-      "imports": [ "dotnet5.4" ]
-    }
+    "netstandard1.6": { }

```

Comment:
@adamralph as far as I know, it just means that if a dependency doesn't explicitly support `netstandard1.6` but supports `dotnet5.4`, use it anyway.